presto-kudu/src/test/java/com/facebook/presto/kudu/TestKuduIntegrationSmoke.java

```diff
@@ -70,6 +73,32 @@ public void testDescribeTable()
         assertEquals(filteredActual, expectedColumns, String.format("%s != %s", filteredActual, expectedColumns));
     }
 
+    @Test
+    public void testShowCreateTable()
+    {
+        queryRunner.execute("CREATE TABLE IF NOT EXISTS test_show_create_table (\n" +
+                "id INT WITH (primary_key=true),\n" +
+                "user_name VARCHAR\n" +
+                ") WITH (\n" +
+                " partition_by_hash_columns = ARRAY['id'],\n" +
+                " partition_by_hash_buckets = 2\n" +
+                ")");
+
+        MaterializedResult result = queryRunner.execute("SHOW CREATE TABLE test_show_create_table");
+        String sqlStatement = (String) result.getOnlyValue();
+        String tableProperties = sqlStatement.split("\\)\\s*WITH\\s*\\(")[1];
+        checkTableProperty(tableProperties, "num_replicas", "\\d+");
+        checkTableProperty(tableProperties, "table_id", "'\\w+'");
+        checkTableProperty(tableProperties, "partition_by_hash_columns", Pattern.quote("ARRAY['id']"));
+        checkTableProperty(tableProperties, "partition_by_hash_buckets", "2");
+    }
+
+    private void checkTableProperty(String tableProperties, String key, String regexValue)
+    {
+        assertTrue(Pattern.compile(key + "\\s*=\\s*" + regexValue + ",?\\s+").matcher(tableProperties).find(),
```

Comment:
you could reuse pattern

Reply:
I don't see how to improve this line. The pattern is created dynamically, as it depends on key and regexValue parameters. What can be shared?